tests/models.json

```diff
@@ -84,6 +87,8 @@
             "embed-multilingual-v3.0",
             "embed-multilingual-light-v3.0"
         ],
-        "text": []
+        "text": [],
+        "image":["dall-e-3",
```

Comment:
Can we remove these image models from cohere. They are not supported

Reply:
Done